### PR TITLE
Update button colours

### PR DESCRIPTION
**Pull request by marianeum**
Got the latest button colours from foundation. Also improved some stories screenshots to avoid unnecessary content

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: app/src/main/java/net/skyscanner/backpack/demo/compose/FloatingNotificationStory.kt [hunk 1]
```diff
@@ -53,46 +53,48 @@ fun FloatingNotificationStory(modifier: Modifier = Modifier) {
         val text = stringResource(R.string.floating_notification_saved)
         val cta = stringResource(R.string.floating_notification_view)
 
-        Column(
-            modifier = Modifier.padding(BpkSpacing.Base),
-            verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
-        ) {
+        val automationMode = LocalAutomationMode.current
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/FloatingNotificationStory.kt:56]: this makes sure the buttons don't appear on the story screenshot which is confusing

File: app/src/main/java/net/skyscanner/backpack/demo/compose/ProgressBarStory.kt [hunk 2]
```diff
@@ -105,20 +106,23 @@ private fun ProgressBarSample(
...
 
-        Row(
-            horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
-        ) {
-            BpkButton(
-                modifier = Modifier.weight(1F),
-                text = stringResource(id = R.string.progress_bar_decrease),
-                onClick = { progress = max(0F, progress - interval) },
-            )
+        val automationMode = LocalAutomationMode.current
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/ProgressBarStory.kt:109]: same here

File: Backpack/src/main/res/values-night/backpack.internal.color.xml [hunk 3]
```diff
@@ -34,20 +34,20 @@
     <color name="__privateButtonLinkOnDarkNormalForeground">#ffffffff</color>
     <color name="__privateButtonLinkOnDarkPressedForeground">#80ffffff</color>
     <color name="__privateButtonLinkPressedForeground">#ffd1f7ff</color>
-    <color name="__privateButtonPrimaryNormalBackground">#ff054184</color>
-    <color name="__privateButtonPrimaryOnDarkDisabledBackground">#ffe0e4e9</color>
-    <color name="__privateButtonPrimaryOnDarkDisabledForeground">#33000000</color>
+    <color name="__privateButtonPrimaryNormalBackground">#ff024daf</color>
```
Comment (marianeum) [inline Backpack/src/main/res/values-night/backpack.internal.color.xml:37]: these are autogenerated based on the latest foundation version

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/internal/BpkButtonColors.kt [hunk 4]
```diff
@@ -32,11 +32,11 @@ internal object BpkButtonColors {
 
     internal val destructiveNormalForeground: Color
         @Composable
-        get() = dynamicColorOf(Color(0xFFC80456), Color(0xFFFF649C))
+        get() = dynamicColorOf(Color(0xFFE70866), Color(0xFFFF649C))
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/internal/BpkButtonColors.kt:35]: also autogenerated

File: app/screenshots/oss/debug/dm/net.skyscanner.backpack.button.BpkButtonTest_disabled.PrimaryOnDark_Standard.png
Comment (xo0ps) [inline app/screenshots/oss/debug/dm/net.skyscanner.backpack.button.BpkButtonTest_disabled.PrimaryOnDark_Standard.png:1]: does it look correct?

File: docs/compose/ProgressBar/screenshots/default.png
Comment (xo0ps) [inline docs/compose/ProgressBar/screenshots/default.png:1]: why are the buttons gone?

Comment (xo0ps) [reply docs/compose/ProgressBar/screenshots/default.png:1]: ok it was intentional

File: app/screenshots/oss/debug/dm/net.skyscanner.backpack.button.BpkButtonTest_disabled.PrimaryOnDark_Standard.png
Comment (marianeum) [reply app/screenshots/oss/debug/dm/net.skyscanner.backpack.button.BpkButtonTest_disabled.PrimaryOnDark_Standard.png:1]: yep the style changed for the button to be dark for the disabled style (which makes sense given that it's for on dark)

**Pull request merged**